osf/external/messages/celery_publishers.py

```diff
@@ -0,0 +1,48 @@
+from kombu import Exchange
+from framework.celery_tasks import app as celery_app
+from framework.celery_tasks.handlers import enqueue_task
+
+
+def publish_deactivated_user(user):
+    enqueue_task(
+        _publish_user_status_change.s(
+            body={
+                'action': 'deactivate',
+                'user_uri': user.url,
```

Comment:
```suggestion
                'user_uri': user.get_semantic_iri(),
```
`OSFUser.url` is just the path `/zguid/` -- let's use a full url instead